packages/react-component-library/src/components/TabSet/TabSet.tsx

```diff
@@ -35,6 +35,21 @@ export const TabSet: React.FC<TabSetProps> = ({
     }
   }
 
+  function handleKeyDown(event: KeyboardEvent<HTMLLIElement>) {
+    const { which } = event
+    let nextIndex
```

Comment:
With the way this function has been written `let nextIndex` then there only needs to be a single `handleClick(nextIndex)`.

It could be easier to read if there was a `getNextIndex(...)` rather than using `let`.